pkg/config/remote/client_test.go

```diff
@@ -108,8 +108,8 @@ func TestClientValidResponse(t *testing.T) {
 
 	targetsKey := generateKey()
 	embeddedRoot := generateRoot(generateKey(), 1, targetsKey)
-	apmConfig := pb.APMSampling{
-		TargetTPS: []pb.TargetTPS{{Service: "service1", Env: "env1", Value: 4}},
+	apmConfig := pbgotrace.APMSampling{
+		TargetTps: []pbgotrace.TargetTPS{{Service: "service1", Env: "env1", Value: 4}},
```

Comment:
Is there a reason to lowercase `TPS` in the field name here? IIRC go codestyle mandates uppercase for acronyms.